build.xml

```diff
@@ -591,6 +591,28 @@
         </javac>
     </target>
 
+    <target name="ci-compile-checked"
```

Comment:
Jenkins already has a deprecations testing check.  Is there a reason to replace it with this check?